### ISSUE TITLE
Deleted conditions

### ISSUE THREAD

**Issue by gert-MORLION** [IEHG/Portrayal-Catalogue#4]
While developing the combined Lookup Tables (hydro and inland specific conditions), some sentences were deleted. Some of these sentences are related to features which are amended in the S-401 FC. This means that it should be reconsidered or at least investigated to reuse these sentences.

**Comment by BerndBirklhuber** [comment]
I agree that the S-101 features that are used in S-401 have to be covered by the LUA scripts. As a first step for the development of the LUA scripts the lines can be added again in the Lookup Tables. 
The official Lookup Tables of edition 2.5 or 2.6 should however only contain features that are part of the FC 2.5 respectively 2.6
